index.js

```diff
@@ -1,30 +1,51 @@
+const multiRet = ['push', 'pop', 'shift', 'unshift']
```

Comment:
Results after changed strings to array:
`let`: 657 bytes
`const`: 665 bytes

Not sure how it does it but I won't sift through the minified output :D

If you say it's OK I'll leave it as `const`